bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/BridgeImpl.java

```diff
@@ -24,7 +24,10 @@
 public class BridgeImpl extends ThingImpl implements Bridge {
 
     private transient List<Thing> things = new CopyOnWriteArrayList<>();
-    private final Logger logger = LoggerFactory.getLogger(BridgeImpl.class);
+
+    private static Logger logger() {
```

Comment:
It should be safe to use transient, see http://stackoverflow.com/a/5889590.
Note that the very same thing is done for the "things" field in the line above as well.